deployer/src/config_manager.py

```diff
@@ -161,12 +160,6 @@ def update_config(self, config_name):
 
             print(message)
 
-            helpers.make_request('/' + config_id, 'PUT',
```

Comment:
(so, the update_config method is not updating config anymore?)

Reply:
It commit and create the right message. Still a time saver :)

Reply:
It also help us to provide the snippet

Reply:
then in another patch, it will be interesting to rename the method to something more accurate